### PR TITLE
Depend on git-remote-hypergit and relay the cli command from package.json

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/noffle/hypergit/issues/4

@noffle I wasn't sure how you handle `package-lock.json` so left it unchanged for now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,14 +29,8 @@ Commands:
...
-The `hypergit` cli lets you manage hypergit remotes. In order to be able to `git
-clone hypergit://...` URLs, you'll need the `git-remote-hypergit` helper:
-
-```
-npm install -g git-remote-hypergit
-```
-
-**TODO**: install this command with the `hypergit` command! (PRs welcome)
+The `hypergit` cli lets you manage hypergit remotes and enables you to `git
+clone hypergit://...` URLs.
```
Comment (ralphtheninja) [inline README.md:33]: I removed mentioning `git-remote-hypergit` as @gozala suggested. Maybe we should have a `## How does it work?` section?

**Comment by ralphtheninja** [comment]
@noffle Also, I'll get my own repo up and submit a PR for that hash.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 > CLI for managing peer-to-peer git repositories
 
+## Install
+
+```
+npm i hypergit -g
+```
+
```
Comment (hackergrrl) [inline README.md:10]: Great idea!

File: README.md (hunk 1 above)
Comment (hackergrrl) [reply README.md:33]: :+1:

File: package.json [hunk 3]
```diff
@@ -7,7 +7,8 @@
     "url": "git://github.com/noffle/hypergit.git"
   },
   "bin": {
-    "hypergit": "bin/cli.js"
+    "hypergit": "bin/cli.js",
+    "git-remote-hypergit": "node_modules/.bin/git-remote-hypergit"
   },
```
Comment (hackergrrl) [inline package.json:12]: Woah! You can do this!? What a clever hack @ralphtheninja :tada:

**Comment by hackergrrl** [comment]
Re package-lock: I don't really use/heed it. As far as I'm concerned it's no more than the price of having to upgrade from npm3. :crying_cat_face:

**Comment by hackergrrl** [comment]
Thank you for this :gift: @ralphtheninja! I tested it and I can `git clone` hypergit URLs just fine.

File: package.json (hunk 3 above)
Comment (vweevers) [reply package.json:12]: This might not work on Windows (but that probably isn't a concern yet)

**Comment by ralphtheninja** [comment]
> Re package-lock: I don't really use/heed it. As far as I'm concerned it's no more than the price of having to upgrade from npm3. crying_cat_face

```
echo package-lock=false >> ~/.npmrc
```

If you _really_ don't care :)

**Comment by hackergrrl** [comment]
Perfect! Thanks.

**Pull request merged**